lib/bundler/audit/database.rb

```diff
@@ -91,15 +94,29 @@ def self.path
       #
       # @since 0.3.0
       #
-      def self.update!
+      def self.update!(quiet)
```

Comment:
I will make this change. I personally prefer simple arguments over option hashes because hashes require extra code to make sure people get a meaningful error when they misspell a hash key (or worse, the library makes no effort to provide an error and silently does the wrong thing).